### ISSUE TITLE
pastedownload.com: popups

### ISSUE THREAD

**Issue by Fatcavill** [uBlockOrigin/uAssets#23915]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://pastedownload.com/`

### Category

popups

### Description

click go after pasting link `https://www.instagram.com/reel/C6hPtvhSWaC/?igsh=YTF6b3NvbHJiazUw`
Pls boost the loading time there and after it load there is 300!! popups

### Other extensions used

no

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.0
Chromium Mobile: 110
filterset (summary):
 network: 149368
 cosmetic: 212841
 scriptlet: 34743
 html: 0
listset (total-discarded, last-updated):
 removed:
  user-filters: null
 added:
  adguard-cookies: 29835-74, now
 default:
  ublock-filters: 38681-667, 52m Δ
  ublock-badware: 8990-2, 52m Δ
  ublock-privacy: 879-0, 52m Δ
  ublock-unbreak: 2310-2, 52m Δ
  ublock-quick-fixes: 87-0, 52m Δ
  adguard-mobile: 9416-11, now
  easylist: 87270-692, 52m Δ
  easyprivacy: 50998-929, 52m Δ
  urlhaus-1: 13355-0, now
  plowe-0: 3729-83, now
filterset (user): [array of 6 redacted]
switchRuleset:
 added: [array of 1 redacted]
userSettings:
 advancedUserEnabled: true
 suspendUntilListsAreLoaded: true
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1411 ms (selfie)
 maxAssetCacheWait: 171 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 3
 network:
  adskeeper.co.uk: 2
  googletagmanager.com: 1
 extended:
  ##.jegulo
  ##a[href^="http://reddit.com/submit"]
  ##a[href^="https://pinterest.com/pin/create/"]
  ##a[href^="https://www.facebook.com/sharer/sharer.php"]
  ##a[href^="whatsapp://send"]
  ##a[href^="http://reddit.com/submit?url="]
  ##a[href*="pinterest.com/pin/create/button"]
  ##a[href^="http://www.tumblr.com/share"]
  ##a[href^="https://twitter.com/home?"]
```
</details>

**Comment by stephenhawk8054** [comment]
The timer is counted at server as well, so boosting the timer on the screen won't change anything, you can test yourself with

```adb
pastedownload.com##+js(nano-sib, innerHTML, *, 0.001)
```

About popup, the multi-popup only occurs if you click on `Download All` which is its purpose? I don't see other popups besides the videos/images?

**Comment by Fatcavill** [comment]
![Screenshot_20240531-141208](https://github.com/uBlockOrigin/uAssets/assets/139833732/1bfb9491-5804-419c-92ad-9c46929888c9)
All three button have popup with `wira.adem.my.id` this link

**Comment by Fatcavill** [comment]
i cant download the video as the popups never stop

**Comment by stephenhawk8054** [comment]
Yes, the `wira.adem.my.id` are the links to redirect to the video link

```
https://wira.adem.my.id/?source=youtube&title=gFrxnn5nOh&type=mp4&download=aHR0cHM6Ly9zY29udGVudC5jZG5pbnN0YWdyYW0uY29tL3YvdDY2LjMwMTAwLTE2LzEyMTMwNzgyM18xODEwMzU3Mzg5NDMzOTExXzg2NzYyMTk2OTQ5OTY1NTU4Mzlfbi5tcDQ%2FX25jX2h0PXNjb250ZW50LmNkbmluc3RhZ3JhbS5jb20mX25jX2NhdD0xMDMmX25jX29oYz1DWHFDRHI0ci04OFE3a052Z0U4dzVtcyZlZG09QVBzMTdDVUJBQUFBJmNjYj03LTUmb2g9MDBfQVlEWDI3S2VLbnFPZmdWMHZnUk43TVY4d2Z4NkNlQmxEc0FZYUMyUk1zWDhGZyZvZT02NjVCMzM4QSZfbmNfc2lkPTEwZDEzYg%3D%3D
```

Blocking them will make the site unable to open the video links.

---

I can't reproduce multiple popups tho. It just opens the `wira.adem.my.id` link and redirects to the video link on my side.

Test again in a new browser profile with only uBO and its default settings.

**Comment by Fatcavill** [comment]
earlier it was just a blank page but now its working
tnx bro

**Comment by Fatcavill** [comment]
no 3 works perfectly thank you bro now its better

**Comment by ghost** [comment]
uBO detected on the site:
<img width="1287" height="123" alt="Image" src="https://github.com/user-attachments/assets/6de2efb9-5775-4fab-baed-dd28d2c12771" />

**Comment by stephenhawk8054** [comment]
Does it affect the video quality at the end?

**Comment by ghost** [comment]
No, it's the same.
